playwright-e2e/lib/component/dsm/table/table.ts

```diff
@@ -0,0 +1,51 @@
+import { Locator, Page } from '@playwright/test';
+import ParticipantPage from 'pages/dsm/participant-page';
+
+export class Table {
+  private readonly _participantPage: ParticipantPage = new ParticipantPage(this.page);
+
+  constructor(private readonly page: Page) {}
+
+  public async openParticipantPageAt(position: number): Promise<ParticipantPage> {
```

Comment:
`position` is row index, right? And, looks like your position starts from 1, not zero. Which means you have to minus 1 when working with Playwright `.nth()` function (example: `.nth(position - 1)`). I'd prefer not doing it.

Reply:
It is already taken into account, you can find it in this function -> `getParticipantAt`;
I did it so because I thought it would be easier for the other developer to select a participant

Reply:
Sounds good. jsdoc would be helpful to explain it's 1-indexed. I prefer zero-indexed because almost all programming languages have been designed this way for arrays, maps, etc. indexing from 0.

Reply:
Yes, I know, but it's Playwright, where we are selecting items from view, where it is the 1, not 0, that's why for a better understanding of which item we are selecting, I decided so. I can change, NP :)

Reply:
Changed